relayer/relays/parachain/types_v2.go

```diff
@@ -58,7 +59,28 @@ type MessageProofV2 struct {
 }
 
 type PendingOrder struct {
-	Nonce       types.U64
-	BlockNumber types.U32
-	Fee         types.U128
+	Nonce       uint64
+	BlockNumber uint32
+	Fee         big.Int
```

Comment:
If you use [types.UCompact](https://github.com/centrifuge/go-substrate-rpc-client/blob/e3b938563803c6a71043c4e7ba2e0d27c400f514/types/ucompact.go#L26) then you won't need to write a custom decoder.